infrastructure/terraform/components/app/variables.tf

```diff
@@ -98,3 +98,9 @@ variable "branch_name" {
   description = "The branch name to deploy"
   default     = "main"
 }
+
+variable "NEXT_PUBLIC_FEATURE_VISIBILITY_TESTING" {
```

Comment:
Lets go with `NEXT_PUBLIC_FEATURE_DISABLE_CONTENT` with values of `true`/`false` and a default of `false`.

Reply:
Done